rest-server/core/src/main/scala/com/actionml/core/template/Dataset.scala

```diff
@@ -37,4 +37,17 @@ abstract class Dataset[T](r: String) extends LazyLogging {
 
 }
 
+
+// allows us to look at what kind of specialized event to create
+case class GenericEvent (
+  //eventId: String, // not used in Harness, but allowed for PIO compatibility
+  event: String,
+  entityType: String,
+  entityId: String,
+  targetEntityId: Option[String] = None,
+  properties: Option[Map[String, Any]] = None,
+  eventTime: String, // ISO8601 date
```

Comment:
maybe use DateTime for eventTime and add iso parser below?

Reply:
not for a scaffold, no data is actually used